moler/device/adbremote.py

```diff
@@ -64,12 +64,15 @@ def __init__(self, sm_params, name=None, io_connection=None, io_type=None, varia
         :param io_constructor_kwargs: additional parameter into constructor of selected connection type
                         (if not given then default one is taken)
         :param initial_state: name of initial state. State machine tries to enter this state just after creation.
+        :param lazy_cmds_events: set False to load all commands and events when device is initialized, set True to load
+                        commands and events when the first time the are required.
```

Comment:
when the first time the are required --> when they are required for the first time

Reply:
Done